x/twap/helpers_test.go

```diff
@@ -0,0 +1,18 @@
+package twap
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/osmosis-labs/osmosis/v10/osmoutils"
+	"github.com/osmosis-labs/osmosis/v10/x/twap/types"
+)
+
+// GetAllHistoricalTimeIndexedTWAPs returns all historical TWAPs indexed by time.
+func (k Keeper) GetAllHistoricalTimeIndexedTWAPs(ctx sdk.Context) ([]types.TwapRecord, error) {
+	return osmoutils.GatherValuesFromStore(ctx.KVStore(k.storeKey), []byte(types.HistoricalTWAPTimeIndexPrefix), []byte(types.HistoricalTWAPTimeIndexPrefix+"a"), types.ParseTwapFromBz)
+}
+
+// GetAllHistoricalPoolIndexedTWAPs returns all historical TWAPs indexed by pool.
+func (k Keeper) GetAllHistoricalPoolIndexedTWAPs(ctx sdk.Context) ([]types.TwapRecord, error) {
+	return osmoutils.GatherValuesFromStore(ctx.KVStore(k.storeKey), []byte(types.HistoricalTWAPPoolIndexPrefix), []byte(types.HistoricalTWAPPoolIndexPrefix+"a"), types.ParseTwapFromBz)
```

Comment:
Can we move these helper methods to `keeper_test.go`

Reply:
They need to be in `twap` package to be defined on keeper correctly. I ended up moving them unexported to `store.go` and then exposing via `export_test.go`. Let me know what you think